recipe/meta.yaml

```diff
@@ -1,22 +1,20 @@
-{% set version = "3.0.0" %}
+{% set version = "3.0.1" %}
 
 package:
   name: cantera-recipe
   version: {{ version }}
 
 source:
-  url: https://github.com/Cantera/cantera/archive/v{{ version }}.tar.gz
-  sha256: 6d17cdef21dae9deaf91351260b68ccf7e264c37c6030c9d4bf26a2cefb2b905
+  url: https://github.com/Cantera/cantera/archive/v3.0.1.tar.gz
+  sha256: 3c51803c2d776e4cb2fed14d5eb17b2fa78a274d0e54d87eb91f23afe3cc4a8c
   patches:
   # https://github.com/conda-forge/cantera-feedstock/pull/13#issuecomment-809382526
   - cross-compile.patch  # [build_platform != target_platform]
   - iomp5-macos.patch  # [osx and not arm64]
   - configure-hdf5.patch  # [build_platform != target_platform]
-  - 1604.diff
-  - ver.diff
 
 build:
-  number: 5
+  number: 7
```

Comment:
Sorry for catching this too late, but I guess this should have been `0`?

Reply:
Yeah, it should have been 🤷‍♂️